src/annotator/guest.ts

```diff
@@ -603,6 +616,11 @@ export class Guest extends TinyEmitter implements Annotator, Destroyable {
    * re-anchoring the annotation.
    */
   async anchor(annotation: AnnotationData): Promise<Anchor[]> {
+    if (this._contentReady) {
+      await this._contentReady;
+      this._contentReady = undefined;
+    }
```

Comment:
nb. If the promise supplied by the host app rejects, there will be an error in the browser console, but annotations will remain visible in the sidebar. So the user can at least see, edit and reply to existing annotations and create new page notes. They obviously won't be able to scroll to annotations in the page.